process.go

```diff
@@ -50,16 +49,20 @@ func (p *Process) execute(ctx context.Context) error {
 }
 
 func (p *Process) pullAndSave(ctx context.Context, subscription *Subscription) error {
-	err := p.subscriber.subscribe(ctx, subscription, func(m *pubsub.Message) error {
+	err := p.subscriber.subscribe(ctx, subscription, func(recvMsg *pubsub.ReceivedMessage) error {
+		m := recvMsg.Message
 		// https://github.com/groovenauts/magellan-gcs-proxy/blob/master/lib/magellan/gcs/proxy/progress_notification.rb#L24
 		msg_id := m.Attributes["job_message_id"]
 		progress, err := strconv.Atoi(m.Attributes["progress"])
 		if err != nil {
-			fmt.Println("Failed to convert %v to int message_id: %v cause of %v", m.Attributes["progress"], msg_id, err)
+			fmt.Printf("Failed to convert %v to int message_id: %v cause of %v", m.Attributes["progress"], msg_id, err)
 			return err
 		}
+		// https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage
+		// A timestamp in RFC3339 UTC "Zulu" format, accurate to nanoseconds. Example: "2014-10-02T15:01:23.045123456Z".
+		time, err := time.Parse(time.RFC3339, m.PublishTime)
```

Comment:
Variable name `time` conflicts with package name `time`, so I think it is better to change variable name (e.g.  `publishTime`)